### PR TITLE
Add part to describe the add of selfsigned certs in Streams trusted c…

### PR DESCRIPTION
**Pull request by benjaminranc**
…erts

Thank you for your contribution to the Axway Streams documentation.

## Describe the changes

Add a part to describe the add of self-signed certificates in the Streams trusted ones.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [x] Does not have conflicts with the base branch
* [x] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [x] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [x] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [x] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [x] Does not contain typos, grammatical errors, etc.
* [x] Does not expose any sensitive information
* [x] Passes the Markdown linter rules (automated via CI check)
* [x] Does not contain broken links

### CODE REVIEW

**Review by lbadenhop — COMMENTED** [review]
Suggest rewording as: "Streams services must have a valid TLS certificate to connect to a TLS endpoint. If your endpoint uses self-signed certificates, you must add them to Streams services as trusted certificates."

**Comment by benjaminranc** [comment]
> Suggest rewording as: "Streams services must have a valid TLS certificate to connect to a TLS endpoint. If your endpoint uses self-signed certificates, you must add them to Streams services as trusted certificates."

The first sentence is not exactly what I meant. I would reword as:
"TLS endpoints to which the Streams services connect must have a valid TLS certificate. If your endpoint uses self-signed certificates, you must add them to Streams services as trusted certificates."

Do you agree?

**Comment by lbadenhop** [comment]
@benjaminranc I agree.

**Comment by andreamussap** [comment]
> @benjaminranc I agree.

@benjaminranc @lbadenhop  have you seen this blog? https://axway.jiveon.com/groups/api-mgt/blog/2020/12/17/axway-open-docs-committing-suggestions

I think that the 'suggest option' would have helped ??

**Comment by lbadenhop** [comment]
@andreamussap Yes, I saw it too late yesterday to incorporate the process into the review.

**Review by lbadenhop — APPROVED** [review]

File: content/en/docs/install/_index.md [hunk 1]
```diff
@@ -344,6 +343,31 @@ kubectl create secret tls streams-ingress-tls-secret --key=${INGRESS_TLS_KEY_PAT
 
 To disable SSL/TLS (not recommended for production use), see [Helm parameters](#helm-parameters).
 
+### Add self-signed TLS certificates
+
+TLS endpoints to which the Streams services connect must have a valid TLS certificate. If your endpoint uses self-signed certificates, you must add them to Streams services as trusted certificates.
```
Comment (allandenis38) [inline content/en/docs/install/_index.md:348]:
```suggestion
TLS endpoints to which Streams services connect must have a valid TLS certificate. If your endpoints uses self-signed certificates, you must add them to Streams services as trusted certificates.
```
